src/core/pageContent/PageContent.tsx

```diff
@@ -87,7 +87,7 @@ export const defaultContentModules = (
           backgroundColor={module.backgroundColor}
           hasLink
           url={module.link.url}
-          imageUrl={module.image.medium}
+          imageUrl={module.image.medium_large}
```

Comment:
Should there be a null check?

Reply:
i am pretty sure i have tested without image. but i ll retest